android/BOINC/app/src/main/java/edu/berkeley/boinc/attach/ProjectAttachService.java

```diff
@@ -119,9 +119,9 @@ private void doUnbindService() {
      * Set credentials to be used in account RPCs.
      * Set / update prior to calling attach or register
      * Saves email and user persistently to pre-populate fields
-     * @param email
-     * @param user
-     * @param pwd
+     * @param email email address of user
+     * @param user user name of user
```

Comment:
I guess `user name` is better because of tautology in this case

Reply:
So the `user` variable should be `userName`?

Reply:
No, I'm talking about the java doc only
`@param user user name` 
is better than 
`@param user user name of user`

Reply:
Ohh, OK.